src/20241118_Multi_AddDHEDGEProtocolToFlashBorrowers/AddDHEDGEProtocolToFlashBorrowers.md

```diff
@@ -0,0 +1,90 @@
+---
+title: "Add dHEDGE Protocol to flashBorrowers"
+author: "Aave Chan Initiative"
+discussions: "https://governance.aave.com/t/arfc-add-dhedge-protocol-to-flashborrowers/19547"
+snapshot: "Direct-to-AIP"
+---
+
+## Simple Summary
+
+The following proposal plans to add dHEDGE Protocol as a whitelisted actor of the Flashborrowers of Aave V3 on Base, Polygon, Arbitrum, and Optimism liquidity pools.
+
+## Motivation
+
+If this proposal is implemented, all flashLoan fees for dHEGDE Protocol using Aave would be waived.
+
+dHEDGE is a tokenized vault protocol that allows users to build, manage, and automate vault strategies. dHEDGE has many vaults that currently use AAVE V3, most notably Toros Finance Leverge tokens.
+
+[dHEDGE Protocol](https://dhedge.org/)[Toros Finance](https://toros.finance/)
+
+Given [[ARFC] Add Contango Protocol, CIAN Protocol and Index Coop to flashBorrowers on Aave v3](https://governance.aave.com/t/arfc-add-contango-protocol-cian-protocol-and-index-coop-to-flashborrowers-on-aave-v3/16478) passed [Snapshot](https://snapshot.org/#/aave.eth/proposal/0x09bb9e7cffc974d330d82ce7a0b0502b573d6f3b4f839ea15d6629613901e96d) with 595k votes in favour, this proposal intends to utilize the Direct-to-AIP proposes to whitelisted permissions on dHEDGE Protocol.
+
+## Specification
+
+Whitelist dHEDGE Protocol as part of FlashBorrowers of Aave V3 on Base, Polygon, Arbitrum & Optimism liquidity pools.
+
+This proposal aims to implement a single AIP, utilizing four similar payloads (one for each network), which will call addFlashBorrower() on the ACL_MANAGER contract for the followings addresses.
+
+**Optimism**
+
+[0x83d1fa384ec44c2769a3562ede372484f26e141b](https://optimistic.etherscan.io/address/0x83d1fa384ec44c2769a3562ede372484f26e141b)
```

Comment:
formatting wise better to use lists for lists as otherwise will be a mess on uis
```suggestion
- [0x83d1fa384ec44c2769a3562ede372484f26e141b](https://optimistic.etherscan.io/address/0x83d1fa384ec44c2769a3562ede372484f26e141b)
```